### PR TITLE
more dualgb related features

### PR DESCRIPTION
**Pull request by Felk**

### CODE REVIEW

**Comment by Felk** [comment]
try this config to disable all features: `"DisabledFeatures": ["Badges", "Cosmetics", "Currencies", "Polls"],`

**Comment by Felk** [comment]
@m4-used-rollout I just spitballed what features need to be disabled. Please take a look. There are some open issues to address:
- [ ] How should incoming subs be handled? Right now it handles them normally. Possible solution: Handle normally but don't emit any overlay events or whispers.
- [ ] How should incoming donations be handled? Right now new core does not handle any donations. That might be a problem.
- [ ] Should the user object fields of the features that are disabled be removed from overlay events? Or is it enough to trust the event overlay to just ignore those? And if yes, is that also a good long-term solution?

**Comment by m4-used-rollout** [comment]
- Handle normally is fine. If people sub and wonder about tokens, we can tell them that they can be used during PBR
- Doesn't the core have to poll for donations? If someone donates while the old core is down, will the donation be processed when the old core comes back up later?
- No, don't worry about limiting the data sent to the overlay. The event input feed should decide what to show, not the core itself.

**Comment by Felk** [comment]
> Handle normally is fine.

Ok good, nothing to do then.

> If someone donates while the old core is down, will the donation be processed when the old core comes back up later?

Hmm, looks like that's what the `refresh_all_tips_worker` does, yes.

> The event input feed should decide what to show, not the core itself.

Good, nothing to do then either.

File: TPP.Core/Configuration/RunmodeConfig.cs [hunk 1]
```diff
@@ -21,5 +22,9 @@ public class RunmodeConfig : ConfigBase, IRootConfig
                      "Inputs can be unmuted at runtime using !unmuteinputs or calling /start_run on the inputserver. " +
                      "Inputs can be muted at runtime using !muteinputs or calling /stop_run on the inputserver. ")]
         public bool MuteInputsAtStartup = false;
+
+        [Description("If not null, this amount of time must pass before a player can switch sides again." +
+                     "Only relevant for dual-sided input profiles.")]
+        public TimeSpan? SwitchSidesCooldown { get; init; } = null;
```
Comment (Felk) [inline TPP.Core/Configuration/RunmodeConfig.cs:28]: hmm... is this required to be updateable at runtime?

Comment (Felk) [reply TPP.Core/Configuration/RunmodeConfig.cs:28]: it is now

**Pull request merged**